phpseclib/Common/Functions/Strings.php

```diff
@@ -33,10 +33,27 @@ class Strings
      * @access public
      * @return string
      */
-    static function shift(&$string, $index = 1)
+    public static function shift(&$string, $index = 1)
     {
         $substr = substr($string, 0, $index);
         $string = substr($string, $index);
         return $substr;
     }
+
+    /**
+     * String Pop
+     *
+     * Inspired by array_pop
+     *
+     * @param string $string
+     * @param int $index
+     * @access public
+     * @return string
+     */
+    public static function pop(&$string, $index = 1)
```

Comment:
When I view the diff on github.com changes to that file are collapsed by default:

![screenshot](https://i.stack.imgur.com/jUPgJ.jpg)

Could be why you're not seeing it?